front/app/containers/InitiativesShow/messages.ts

```diff
@@ -132,4 +132,17 @@ export default defineMessages({
     id: 'app.containers.InitiativesShow.cosponsorCTA',
     defaultMessage: 'Cosponsor this proposal',
   },
+  manageInvitationsLinkText: {
+    id: 'app.containers.InitiativesShow.manageInvitationsLinkText',
+    defaultMessage: 'Manage cosponsorship invitations',
```

Comment:
It seems we never used the word `cosponsorship` before. Could we use `cosponsor` here too if it makes sense?

Not a big deal though if you think `cosponsorship` makes more sense here.

Reply:
I think that's better English. E.g. "fellowship invitation" rather than "fellow invitation". You invite someone (a cosponsor) for something (cosponsorship).

I would be OK with cosponsors, but translations have already happened in some languages so keeping it.